images/extension-builders/entrypoint.sh

```diff
@@ -79,6 +79,11 @@ command_build()  {
 		shift
 	done
 
+	# Since code import in #98, this was always required.
+	if [ "${output_file}" == "" ]; then
```

Comment:
fyi we would never reach this branch as long as we use the container through `getenvoy extension init` https://github.com/tetratelabs/getenvoy/blob/master/pkg/extension/workspace/toolchain/builtin/default_config.go#L69-L77

Reply:
*build

Reply:
yeah I was wondering how much we should support workspaces init'ed in the past... if not an issue, we should delete this I agree. wdyt?

Reply:
not an issue, so let's delete 😄

Reply:
sure